src/pretalx/agenda/templates/agenda/schedule_list.html

```diff
@@ -0,0 +1,92 @@
+{% extends "agenda/base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block agenda_custom_header %}
+    <script src="{% static "agenda/js/offline.js" %}"></script>
+{% endblock %}
+
+{% block agenda_content %}
+<span id="offline-vars" data-event="{{ request.event.urls.schedule }}"></span>
+{% if data|length > 1 %}
+<div id="navigator">
+  <i class="fa fa-dot-circle-o"></i> nav
+  <div class="days">
+    {% for day in data %}
+    <a href="#{{ day.start.date|date:"c" }}">{{ day.start.date|date }}</a>
+    {% endfor %}
+  </div>
+</div>
+{% endif %}
+
+{% if request.event.settings.use_tracks and request.event.tracks.count > 1 %}
+<div id="legend">
+  <i class="fa fa-dot-circle-o"></i> ?
+  <ul class="tracks">
+    {% for track in request.event.tracks.all %}
+    <li style="color: {{ track.color }}">{{ track.name }}</li>
+    {% endfor %}
+  </ul>
+</div>
+{% endif %}
+
+<div id="fahrplan">
```

Comment:
This whole document is very close to the regular schedule, so extracting shared blocks to templates that can be included would be better.